pkg/gui/confirmation_panel.go

```diff
@@ -48,27 +48,15 @@ func (gui *Gui) wrappedPromptConfirmationFunction(handlersManageFocus bool, func
 	}
 }
 
-func (gui *Gui) closeConfirmationPrompt(handlersManageFocus bool) error {
+func (gui *Gui) deactivateConfirmationPrompt() {
 	gui.Mutexes.PopupMutex.Lock()
 	gui.State.CurrentPopupOpts = nil
 	gui.Mutexes.PopupMutex.Unlock()
 
-	// we've already closed it so we can just return
```

Comment:
this was likely only here from back when we ran the confirmation prompt's handler _before_ closing the prompt. We should be able to unconditionally call PopContext (as we're now doing above) without any concerns about what we're popping